test/e2e_node/remote/remote.go

```diff
@@ -207,22 +207,31 @@ func getTestArtifacts(host, testDir string) error {
 	if err := os.MkdirAll(logPath, 0755); err != nil {
 		return fmt.Errorf("failed to create log directory %q: %w", logPath, err)
 	}
-	// Copy logs to artifacts/hostname
-	if _, err := runSSHCommand(host, "scp", "-r", fmt.Sprintf("%s:%s/results/*.log", GetHostnameOrIP(host), testDir), logPath); err != nil {
-		return err
+	// Copy logs (if any) to artifacts/hostname
+	if _, err := SSH(host, "ls", fmt.Sprintf("%s:%s/results/*.log", GetHostnameOrIP(host), testDir)); err == nil {
+		if _, err := runSSHCommand(host, "scp", "-r", fmt.Sprintf("%s:%s/results/*.log", GetHostnameOrIP(host), testDir), logPath); err != nil {
+			return err
+		}
 	}
 	// Copy json files (if any) to artifacts.
 	if _, err := SSH(host, "ls", fmt.Sprintf("%s/results/*.json", testDir)); err == nil {
 		if _, err = runSSHCommand(host, "scp", "-r", fmt.Sprintf("%s:%s/results/*.json", GetHostnameOrIP(host), testDir), *resultsDir); err != nil {
 			return err
 		}
 	}
+	// Copy junit results (if any) to artifacts
 	if _, err := SSH(host, "ls", fmt.Sprintf("%s/results/junit*", testDir)); err == nil {
 		// Copy junit (if any) to the top of artifacts
 		if _, err = runSSHCommand(host, "scp", fmt.Sprintf("%s:%s/results/junit*", GetHostnameOrIP(host), testDir), *resultsDir); err != nil {
 			return err
 		}
 	}
+	// Copy container logs to artifacts/hostname
+	if _, err := SSH(host, "chmod", "-R", "a+r", "/var/log/pods"); err == nil {
```

Comment:
Ensure the container/pod logs are readable before we run `scp`, if the chmod runs fine, we can just recursively copy over everything below it.